content/docs/use-cases/experiment-bookkeeping.md

```diff
@@ -0,0 +1,49 @@
+# Data Science Experiment Bookkeeping
+
+Iterating on data science implies a large number of attempts at tuning
+parameters, improving code, trying different data, and collecting metrics, for
+example when optimizing a machine learning model's accuracy. Keeping track of
+all these <abbr>experiments</abbr> is challenging but essential, as we'll need
+to compare, visualize, share, and _productionize_ the results. We also want to
+be able to reconnect any recorded metrics back to their original setup.
+
+![](/img/lose-experimentation.png) _Loose experimentation_
+
+DVC provides a layer of
+[experiment management](/doc/user-guide/experiment-management) features to take
+automatic snapshots of any variations in your project, and handle them with
+simple terminal [commands](/doc/command-reference/exp). Some things you can do
+include:
+
+- Rapid iteration on experiment ideas by capturing variations from a baseline
+  with a simple command (`dvc exp run`).
+- Automatic bookkeeping of changes to data dependencies, code,
```

Comment:
> @dberenbaum dberenbaum 12 days ago Member
A separate use case for orchestration makes sense

Updated https://github.com/iterative/dvc.org/issues/2544 to 

> List of things that DVC could do can be simplified (pretty much should be clear that it's a full featured experiment tracker and execution engine).

This is still a pretty valid comment for the current doc though, and I simplified the list a little in 398afa86. I think it reflects that idea now, but I'm not sure it can be much shorter). PTAL